cncserver.js

```diff
@@ -1125,6 +1126,18 @@ function serialPortReadyCallback() {
       return;
     }
 
+    // Setting the value of the power to the pen
+    if (typeof inPen.power !== "undefined") {
+      powers = botConf.get('penpower');
+      if(typeof powers === "undefined") { // We have no super powers
+        powers = {min: 0, max: 0};  //Set the powers to zero

```

Comment:
Forgot the space before your comment... Tiny nit pick, I know :no_pedestrians: :ant: